packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -326,6 +326,13 @@ class WebKitWebViewController extends PlatformWebViewController {
   Future<void> addJavaScriptChannel(
     JavaScriptChannelParams javaScriptChannelParams,
   ) {
+    final String channelName = javaScriptChannelParams.name;
+    if (_javaScriptChannelParams.containsKey(channelName)) {
+      throw ArgumentError(
+        'A JavaScriptChannel with name `$channelName` has already been added.',
```

Comment:
nit: this sounds like a success message. how about "A JavaScriptChannel with name `$channelName` already exists?"